nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/src/main/java/org/apache/nifi/snmp/processors/ListenTrapSNMP.java

```diff
@@ -65,14 +74,32 @@ public class ListenTrapSNMP extends AbstractSessionFactoryProcessor {
             .addValidator(StandardValidators.PORT_VALIDATOR)
             .build();
 
-    public static final PropertyDescriptor SNMP_USM_USERS_FILE_PATH = new PropertyDescriptor.Builder()
```

Comment:
I followed the naming pattern of another processor. I think JSON in the filename is more meaningful. I can rename it to JSON_FILE_PATH if you think so.

Reply:
I see. If it's not problem to you I'd rename it to JSON_FILE_PATH then because the user has json file and json option and from the developer's point of view it is easy to get confused.